worldengine/model/world.py

```diff
@@ -132,11 +202,9 @@ def _from_protobuf_quantiles(p_quantiles):
 
     @staticmethod
     def _from_protobuf_matrix_with_quantiles(p_matrix):
```

Comment:
I think I mentioned this recently, but how about putting all the Protobuf-stuff into its own file while sticking as closely as possible to how things are done in the HDF5-file? The saving of both formats is extremely similar, and all the Protobuf-code is cluttering up this file a bit.